src/main/java/org/threeten/extra/scale/UtcInstant.java

```diff
@@ -196,12 +202,12 @@ public static UtcInstant of(TaiInstant instant) {
     //-------------------------------------------------------------------------
     /**
      * Obtains an instance of {@code UtcInstant} from a text string
-     * {@code 2007-12-03T10:15:30.00Z}.
+     * (e.g. {@code 2007-12-03T10:15:30.00Z}).
```

Comment:
Using "e.g." can confuse Javadoc because the dots make it think a sentence has ended. Use ", such as {@code...}." instead

Reply:
Resolved